### DISCUSSION TITLE
Where does the elasticsearch external link show up in the UI?

### DISCUSSION THREAD

**Discussion by whitleykeith** [apache/airflow#16759] [Q&A]
I configured my Airflow with the elasticsearch.frontend property that is supposed to allow users to get redirected straight to something like Kibana. However, I can seem to find the actual link in the UI and the normal log page doesn't redirect me. 

Am I blind or is there something I'm missing? I'm on Airflow 2.1

**Comment by Trolldemorted** [comment]
Did you manage to get it work somehow?

**Comment by M4C4R** [comment]
I'm on Airflow 2.10.5, you click the blue task number in the **Logs** tab.

---
<kbd>
<img width="1249" height="314" alt="image" src="https://github.com/user-attachments/assets/4ee6e66f-7633-41a3-86eb-8933697a7d5a" />
</kbd>

**Reply by M4C4R** [reply]
No idea where it is in Airflow 3.X though...

UPDATE: It was never there, Airflow 3.2.0 re-introduces this
<kbd>
<img width="1249" height="314" alt="image" src="https://github.com/user-attachments/assets/dc9c7e8b-4ea8-4479-833b-779f85da73e7" />
</kbd>
